docs/README.md

```diff
@@ -103,6 +114,18 @@ const App = () =>{
   );
 }
 ```
+### Parameters
+1. `className` (_any_) : Initial value of the state.
+
```

Comment:
The classname should have a type of string and. It is the class to be added to the body when it is dark mode